src/node-capnp/capnp.d.ts

```diff
@@ -41,6 +41,7 @@ declare module Capnp {
   function parse<Builder, Reader>(type: StructSchema<Builder, Reader>, buffer: Buffer): Reader;
   function serialize<Builder, Reader>(type: StructSchema<Builder, Reader>, builder: Builder): Buffer;
   function connect(addr: string): Connection;
+  function connect<Client>(addr: string, clientBootstrapCapability: Client): Connection;
   function importFile<SchemaBag>(path: string): SchemaBag;
   function importSystem<SchemaBag>(path: string): SchemaBag;
```

Comment:
Is there any advantage to using a generic parameter here, rather than typing `clientBootstrapCapability` as `object`?

Scope creep: I've felt similarly about `importFile` and `importSystem` for a while. The problem is more severe in their case, because their return value will implicitly cast to any type, when they could force the caller to write an explicit type assertion instead.